indico/modules/events/registration/templates/display/_registration_summary_blocks.html

```diff
@@ -4,11 +4,11 @@
             <td colspan="2">{{ section.title }}</td>
         </tr>
         {% for field in section.children if field.is_field and field.id in registration.data_by_field -%}
-            <tr class="{{ 'deleted-field' if from_management and field.is_deleted }}">
+            <tr class="{{ 'deleted-field' if from_management and (field.is_deleted or field.parent.is_deleted) }}">
```

Comment:
You could use `section.is_deleted` - since you iterate over `section.children` that's always the parent, but by using the section you avoid possibly triggering a relationship load